src/main/webapp/X-workz-java-testing-development-software-training-center.html

```diff
@@ -1,348 +1,356 @@
-											<div class="check">
-												<div class="inside"></div>
-											</div></li>
-									</ul>
-									<div class="clearfix"></div>
-								</div>
-								<input type="submit" value="Submit">
-							</form>
-						</div>
-					</div>
-				</div>
-			</div>
-		</div>
-	</div>
-
-
-	<div class="footer-agileits-w3layouts">
-		<div class="container">
-			<div class="btm-logo-w3ls">
-				<h2>
-					<a href="index.html"> <img src="assets/images/Logo.png"
-						height="100px" width="220px" />
-					</a>
-				</h2>
-			</div>
-			<div class="subscribe-w3ls">
-				<h6>Let us inform you about everything important directly.</h6>
-				<form action="#" method="post">
-					<div class="col-md-8 col-sm-8 col-xs-8 input-flds-w3-agile">
-						<input type="email" name="Email" placeholder="Email"
-							required="true" id="emailSubField">
-					</div>
-					<div class="col-md-4 col-sm-4 col-xs-4 input-flds-w3-agile">
-						<input type="button" value="Subscribe" id="subscribe"
-							class="subscribeForm">
-						<button type="submit" hidden="true" id="submit">submit</button>
-					</div>
-					<div class="clearfix"></div>
-				</form>
-			</div>
-			<div class="social-icons-agileits">
-				<ul>
-					<li><a
-						href="https://www.facebook.com/xworkzdevelopmentcenter/"><span
-							class="fa fa-facebook"></span></a></li>
-					<li><a href="https://twitter.com/workz_x"><span
-							class="fa fa-twitter"></span></a></li>
-				</ul>
-			</div>
-
-			<div class="clearfix"></div>
-		</div>
-	</div>
-	<div class="copyright-w3layouts">
-		<div class="container">
-			<p>&copy; 2019 X-Workz . All Rights Reserved</p>
-		</div>
-	</div>
-
-	<a href="#home" class="scroll" id="toTop" style="display: block;">
-		<span id="toTopHover" style="opacity: 1;"> </span>
-	</a>
-
-	<!-- Modal -->
-	<div class="modal fade " id="notification" role="dialog">
-		<div class="modal-dialog">
-
-			<!-- Modal content-->
-			<div class="modal-content">
-				<div class="modal-header">
-					<button type="button" class="close" data-dismiss="modal">&times;</button>
-						<h4 class="modal-title">X-Workz<sub>(ODC)</sub></h4>
-				</div>
-				<div class="modal-body notData">
-
-
-				</div>
-				<div class="modal-footer">
-					<button type="button" class="btn btn-default" data-dismiss="modal">
-						Close</button>
-				</div>
-			</div>
-
-		</div>
-	</div>
-
-	<!-- End Modal -->
-	<script type="text/javascript" src="js/mainJS.js"></script>
-
-</body>
-
-</html>
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+<!-- Page title and description contents are displayed in search results -->
+<title>Software Development Training Center-learn java development with Certification</title>
+<meta name="description=" content="X-workz is software training center ,we train the students on
+java development , Testing ,Web development. We also offer free classes for core java , we train on full-stack development with Enterprise Application development">
```

Comment:
mention more technologies angular,sql, web , selenium , projects, internships...college placements etc,,,,

Reply:
used in different pages sir